test/e2e/adminapi_redeployvm.go

```diff
@@ -67,19 +68,18 @@ var _ = Describe("[Admin API] VM redeploy action", func() {
 			var nodeNotReady, rebooted, nodeReady bool
 
 			for _, event := range events.Items {
-				if event.CreationTimestamp.After(nodeKillTime.Time) {
-					if !nodeNotReady &&
-						event.Reason == "NodeNotReady" &&
-						event.Regarding.Name == *vm.Name {
+				if event.CreationTimestamp.After(nodeKillTime.Time) && event.Regarding.Name == *vm.Name {
```

Comment:
I've moved a repeat conditioin of `event.Regarding.Name == *vm.Name` into this parent conditional